src/parse.c

```diff
@@ -10480,15 +10533,118 @@ static tree_t p_psl_clock_declaration(void)
    psl_node_t p = psl_new(P_CLOCK_DECL);
    psl_set_tree(p, expr);
 
+   tree_set_ident(parent, well_known(W_DEFAULT_CLOCK));
    consume(tSEMI);
 
    psl_set_loc(p, CURRENT_LOC);
    psl_check(p);
 
+   return p;
+}
+
+static psl_node_t p_psl_property_declaration(tree_t parent)
+{
+   // property PSL_Identifier [ ( Formal_Parameter_List ) ] is Property ;
+
+   assert(tree_kind(parent) == T_PSL);
+
+   BEGIN("PSL property declaration");
+
+   scan_as_psl();
+
+   psl_node_t decl = psl_new(P_PROPERTY_DECL);
+   consume(tPROPERTY);
+
+   scan_as_vhdl();
+   tree_set_ident(parent, p_identifier());
+   scan_as_psl();
+
+   if (optional(tLPAREN)) {
+      p_psl_formal_parameter_list(parent, true);
+      consume(tRPAREN);
+   }
+
+   consume(tIS);
+
+   psl_node_t prop = p_psl_property();
+   psl_set_value(decl, prop);
+
+   consume(tSEMI);
+
+   psl_set_loc(decl, CURRENT_LOC);
+   psl_check(decl);
+
+   scan_as_vhdl();
+
+   return decl;
+}
+
+static psl_node_t p_psl_sequence_declaration(tree_t parent)
+{
+   // sequence PSL_Identifier [ ( Formal_Parameter_List ) ] is Sequence ;
+
+   assert(tree_kind(parent) == T_PSL);
+
+   BEGIN("PSL sequence declaration");
+
+   scan_as_psl();
+
+   psl_node_t decl = psl_new(P_SEQUENCE_DECL);
+   consume(tSEQUENCE);
+
+   scan_as_vhdl();
+   tree_set_ident(parent, p_identifier());
+   scan_as_psl();
+
+   if (optional(tLPAREN)) {
+      p_psl_formal_parameter_list(parent, false);
+      consume(tRPAREN);
+   }
+
+   consume(tIS);
+
+   psl_node_t prop = p_psl_sequence();
+   psl_set_value(decl, prop);
+
+   consume(tSEMI);
+
+   psl_set_loc(decl, CURRENT_LOC);
+   psl_check(decl);
+
+   scan_as_vhdl();
+
+   return decl;
+}
+
+static tree_t p_psl_declaration(void)
+{
+   // PSL_Declaration ::=
+   //      Property_Declaration
```

Comment:
```suggestion
   // Property_Declaration
```

I normally leave off the first part of the grammar production in the comments.